libminifi/include/SchedulingAgent.h

```diff
@@ -39,6 +41,9 @@
 #include "core/controller/ControllerServiceProvider.h"
 #include "core/controller/ControllerServiceNode.h"
 
+#define SCHEDULING_WATCHDOG_CHECK_PERIOD 1000  // msec
+#define SCHEDULING_WATCHDOG_ALERT_PERIOD 5000  // msec
```

Comment:
`SCHEDULING_WATCHDOG_ALERT_PERIOD` is now redundant with the introduction of `nifi.flow.engine.alert.time`

Reply:
That should be the default value to avoid hardcoded numbers in the code.
Fixed, renamed.